stable_baselines3/common/vec_env/vec_frame_stack.py

```diff
@@ -9,19 +9,24 @@
 
 class VecFrameStack(VecEnvWrapper):
     """
-    Frame stacking wrapper for vectorized environment
+    Frame stacking wrapper for vectorized environment.
+    Frames are concatenated on desired axis (e.g. 4 RGB images -> 12 channels).
 
     :param venv: the vectorized environment to wrap
     :param n_stack: Number of frames to stack
+    :param channels_first: If True, stack on first image dimension, otherwise stack on the last (default)
     """
 
-    def __init__(self, venv: VecEnv, n_stack: int):
+    def __init__(self, venv: VecEnv, n_stack: int, channels_first: bool = False):
```

Comment:
why not re-using you helper?
and maybe have a parameter `channels_order` that is optional and that can overwrite the result of the `is_channel_first()` helper?